vyper/semantics/types/module.py

```diff
@@ -79,7 +80,7 @@ def abi_type(self) -> ABIType:
         return ABI_Address()
 
     def __str__(self):
-        return self._id
+        return Path(self._id).stem
```

Comment:
what if the type is aliased? e.g.
```
from ethereum.ercs import IERC20 as BROWNBEAR
```

Reply:
right, that would show the ierc20

Reply:
Do you want it to show the alias when calling str? Or in one of the two issues https://github.com/vyperlang/vyper/issues/4237 or https://github.com/vyperlang/vyper/issues/4237?